zulipterminal/themes/dracula.py

```diff
@@ -0,0 +1,88 @@
+"""
+DRACULA
+-------
+This theme uses the official dracula color scheme.
+For color reference see:
+    https://github.com/dracula/vim/blob/d1864ac0734ce51150affa96a35a1e01ade08b79/colors/dracula.vim
+
+For further details on themefiles look at the theme contribution guide
+"""
+from enum import Enum
+
+from zulipterminal.config.color import color_properties
+
+
+# fmt: off
+class DraculaColor(Enum):
+    # color          =  16code          256code   24code
+    DEFAULT          = 'default         default   default'
+    BG_DARK          = 'dark            h235      #21222C'
+    BG_LIGHT         = 'white           h253      #F8F8F2'
+    COMMENT          = 'brown           h61       #6C72A4'
+    SELECTION        = 'dark_blue       h239      #44475A'
+    SUBTLE           = 'blue            h238      #424450'
+    CYAN             = 'cyan            h117      #8BE9FD'
+    GREEN            = 'green           h84       #50FA7B'
+    ORANGE           = 'orange          h215      #FFB86C'
+    PINK             = 'pink            h212      #FF79C6'
+    PURPLE           = 'magenta         h141      #BD93F9'
+    RED              = 'red             h203      #FF5555'
+    YELLOW           = 'yellow          h228      #F1FA8C'
+
+Color = color_properties(DraculaColor, 'BOLD')
+
+
+STYLES = {
+    # style_name       :  foreground                   background
+    None               : (Color.BG_LIGHT,              Color.BG_DARK),
+    'selected'         : (Color.BG_DARK,               Color.BG_LIGHT),
+    'msg_selected'     : (Color.BG_DARK,               Color.BG_LIGHT),
```

Comment:
I'm not sure this color combination works OK when there are highlighted messages with complicated content - or perhaps the message content styles could be adjusted.